modules/update-conditional-updates.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-cli.adoc
+// * updating/understanding-upgrade-channels-releases.adoc
+
+[id="update-conditional-upgrade-path{context}"]
+= Updating to a conditional upgrade path
+
+You can update a conditional upgrade path using the OpenShift CLI (`oc`) 4.10 or later.
+
+.Procedure
+
+. You can perform an upgrade, by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to <version> <.>
+----
+<.> `<version>` is the update version you want to update to.
++
+The recommended updates are also available on the web console from the Administrator perspective.
+When updates are not recommended because a risk might apply you can view the view the description of the update.
++
+To view the description of the update, run the following command:
```

Comment:
Possibly worth simplifying to:

1. When updates are not recommended because a risk might apply, view the description of the update:
    $ oc adm upgrade --include-not-recommended

Reply:
There is a convention to use ", run the following command." in docs. So, I'm making an update as recommended with the convention.